### ISSUE TITLE
[JENKINS-33930] new tabbed job config page broken accessibility

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-06-a2#1460]
<p>you can no longer use the keyboard to navigate on the restyled "FreeStyle" job configuration page.</p>

<p>There is no way to change the tabs which would confuse a user (they are not tabs - but look like tabs? - when an item gets focus via a tab key the "tab" thing should change highlight as it does if you just scroll<br/>
The tabbing order of the items and help is completely out of whack.<br/>
The red X on a build step never seems to receive focus</p>

**Attachments:**

- [Screenshot 2016-05-30 10.59.05.png](https://issues.jenkins.io/secure/attachment/32827/Screenshot 2016-05-30 10.59.05.png)

---
<details><summary><i>Originally reported by <a title="James Nord" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=teilo">teilo</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-33930" target="_blank">new tabbed job config page broken accessibility</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a title="Manuel Recena Soto" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=recena">recena</a>
<li><b>reported by</b>: teilo
<li><b>status</b>: Resolved
<li><b>priority</b>: Critical
<li><b>resolution</b>: Cannot Reproduce
<li><b>resolved</b>: 2018-01-24T22:08:07+00:00
<li><b>votes</b>: 1
<li><b>watchers</b>: 8
<li><b>imported</b>: 2025-11-06
</ul></i>
</details>

**Comment by jenkins-infra-bot** [comment]
<i><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck">danielbeck</a>:</i>
<p>Is this issue complete? Last char in the description is a comma.</p>

**Comment by jenkins-infra-bot** [comment]
<i><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck">danielbeck</a>:</i>
<blockquote><p>The tabbing order of the items and help is completely out of whack.</p></blockquote>

<p>Not a regression from 1.x. Order there is browser UI/chrome, then help, then page navigation/header (which is mostly gone from 2.x), then form elements, then footer, and it's still the same in 2.0-beta-1.</p>

**Comment by jenkins-infra-bot** [comment]
<i><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=teilo">teilo</a>:</i>
<p>Was partially fixed by <a href="https://github.com/jenkinsci/jenkins/pull/2227" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jenkins/pull/2227</a> (delete not showing focus)<br/>
Tab seems order is fixed<br/>
Tab selecting UX issue(s) remain</p>
<ul class="alternate" type="square">
	<li>Can not change them with tab</li>
	<li>Highlighted tab is not for the section you are editing/has focus (but is the same now if you use the mouse - the tab highlight matches what is at the top of the screen)</li>
</ul>
